src/chttpd_handlers.erl

```diff
@@ -51,6 +54,20 @@ design_handler(HandlerKey, DefaultFun) ->
 %% Internal Function Definitions
 %% ------------------------------------------------------------------
 
+is_disabled(KeyBin) ->
+    BlacklistConfig = config:get("chttpd", "url_handler_blacklist", []),
```

Comment:
@kxepal Well, configuration is part of what the end-user sees, so we can’t just fix it later. We need to consider an upgrade plan and everything. I’d like to avoid doing that and I don’t mind spending another day of nailing this one shut so we can forget about it.

But you make a good point, I’ll keep thinking about this.